### PR TITLE
feat: Add Rose Pine themes

### PR DESCRIPTION
**Pull request by wilkinson4**
## Rose Pine
rosePine

## Rose Pine Moon
rosePineMoon

## Rose Pine Dawn
rosePineDawn

### CODE REVIEW

**Comment by joshgoebel** [comment]
Please provide a few snaps to show us what it looks like.

**Comment by wilkinson4** [comment]
> Please provide a few snaps to show us what it looks like.

Screenshots added. Thank you!

**Comment by wilkinson4** [comment]
@joshgoebel Is this good to go, or something I can fix possibly?

**Comment by joshgoebel** [comment]
Please add an entry to the changelog.

File: CHANGES.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Developer Tools:
 
 Themes:
 
-- Nothing yet.
+- Added `Rosé Pine` theme [William Wilkinson][]
```
Comment (joshgoebel) [inline CHANGES.md:23]: Need to add a link as well line 30, etc.

Comment (wilkinson4) [reply CHANGES.md:23]: Oops 😂 , done.

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**